test-case-reporting/app.ts

```diff
@@ -128,7 +131,10 @@ app.get('/test-results/history/:testCaseId', async (req, res) => {
       res.send(
         render('test-run-list', {
           title: `Test Runs for test case "${testCaseName}"`,
-          testRuns,
+          testRuns: testRuns.map((testRun, index) => ({
+            testRun,
+            index,
+          })),
```

Comment:
Nit: I suspect this could fit cleanly on one line, which in this case would be less visual clutter. If not, nbd. But add a comment explaining why this is needed, and consider a simple helper like
```javascript
function enumeratedTestRuns(testRuns) {
  return testRuns.map((testRun, index) => ({testRun, index});
}
```

You'll end up reusing this for the flaky-sorted list view as well.

Reply:
Done.